README.md

```
@@ -1,65 +1,80 @@
+- Dimension: 1536
+- Metric: dotproduct
+- Spec: aws, us-east-1
+- Namespace: `default`
+
+## <a name="license">📝 License</a>
+
+This project is under the MIT license. See the [LICENSE](LICENSE) for details.
+
+> Credit to Fellipe Utaka for the [Discord Bot Template](https://github.com/fellipeutaka/discord-bot-template)
+
+[pr]: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "pr"

(MD053, link-image-reference-definitions)